### DISCUSSION TITLE
Can't install the library into IDE 2.0

### DISCUSSION THREAD

**Discussion by AdamHoskins** [tttapa/Control-Surface#805] [Q&A]
I can't seem to install the library into IDE 2.0.

I have tried installing from zip, it copies it into my library folder but it doesn't appear in the IDE. I tried chaning the dashes to underscores, that doesn't help. 

Any ideas?

**Comment by tttapa** [comment]
I can't seem to be able to reproduce the issue. I followed [these instructions](https://tttapa.github.io/Control-Surface-doc/Doxygen/d8/da8/md_pages_Installation.html) with the latest version IDE (2.0.0) and after restarting the IDE, it showed up as expected:

![Screenshot from 2022-09-24 23-35-07](https://user-images.githubusercontent.com/21296729/192119645-3d5480ef-1b5b-432c-ab49-c6330e2c83fa.png)

**Comment by AdamHoskins** [comment]
Moving my arduino folder off my One Drive and onto a root folder fixed it
